test/bdd/features/datalayer.feature

```diff
@@ -236,4 +236,81 @@ Feature: Data layer related features
     Given DC initiates the replication for last imported dataset
     And I wait for replications to finish
     Given DV publishes query consisting of path: "identifiers.id", value: "urn:epc:id:sgtin:Batch_1" and opcode: "EQ" to the network
-    Then all nodes with last import should answer to last network query by DV
\ No newline at end of file
+    Then all nodes with last import should answer to last network query by DV
+
+
+
+  @third
+  Scenario: Cover message routing protocol via proxy nodes
+    Given the replication difficulty is 0
+    And I setup 10 nodes
+    And I override configuration for 3rd node
```

Comment:
Why don't you use
I override configuration for all nodes - to override configuration on all nodes
Also
I start the nodes - to start all nodes

Reply:
Since the configuration isn't applied to all nodes, it can't be done with a global step.
Sometimes it's important in which order the nodes start, the trail BDD tests have the same limitation since we rely on the ordering for determining which node has which ERC-725, which is used for connectors

Reply:
I agree with Kuki. We need to start proxy nodes first, then we can start one part after another one.